doc/sphinx/source/recipes/recipe_ipccwg1ar6ch3.rst

```diff
@@ -500,17 +527,35 @@ Example plots
 .. figure::  /recipes/figures/ipccwg1ar6ch3/fig_3_42_a.png
    :align:   center
 
-   Figure 3.42a: Relative space–time root-mean-square deviation (RMSD)
+   Figure 3.42a: Relative space-time root-mean-square deviation (RMSD)
    calculated from the climatological seasonal cycle of the CMIP simulations
-   (1980–1999) compared to observational datasets. A relative performance
+   (1980-1999) compared to observational datasets. A relative performance
    measure is displayed, with blue shading indicating better and red shading
    indicating worse performance than the median error of all model results. A
    diagonal split of a grid square shows the relative error with respect to the
    reference data set (lower right triangle) and an additional data set (upper
    left triangle). Reference/additional datasets are from top to bottom in (a):
-   ERA5/NCEP-NCAR-R1, GPCP-SG/GHCN, CERES-EBAF, CERES-EBAF, CERES-EBAF,
-   CERES-EBAF, JRA-55/ERA5, ESACCI-SST/HadISST, ERA5/NCEP-NCAR-R1,
-   ERA5/NCEP-NCAR-R1, ERA5/NCEP-NCAR-R1, ERA5/NCEP-NCAR-R1, ERA5/NCEP-NCAR-R1,
-   ERA5/NCEP-NCAR-R1, AIRS/ERA5, ERA5/NCEP-NCAR-R1. White boxes are used when
-   data are not available for a given model and variable. Figure is updated 
-   and expanded from Bock et al. (2020).
+   ERA5/NCEP, GPCP-SG/GHCN, CERES-EBAF, CERES-EBAF, CERES-EBAF, CERES-EBAF,
+   JRA-55/ERA5, ESACCI-SST/HadISST, ERA5/NCEP, ERA5/NCEP, ERA5/NCEP, ERA5/NCEP,
+   ERA5/NCEP, ERA5/NCEP, AIRS/ERA5, ERA5/NCEP. White boxes are used when data
+   are not available for a given model and variable. Figure is updated and
+   expanded from Bock et al. (2020).
+
+.. figure::  /recipes/figures/ipccwg1ar6ch3/patterncor.png
+   :align:   center
+
+   Figure 3.43 | Centred pattern correlations between models and observations
+   for the annual mean climatology over the period 1980-1999. Results are
+   shown for individual CMIP3 (green), CMIP5 (blue) and CMIP6 (red) models (one
```

Comment:
In the last test run I made, it looks more like "CMIP6 (green), CMIP5 (blue) and CMIP3 (red) models" from left to right. Could you please maybe check if the order of CMIP phases should be reversed in this caption or in the plot? You can find my test run in `/work/bd0854/b309192/runs/tests/PR_2772/esmvaltool_output/recipe_ipccwg1ar6ch3_fig_3_43_20230214_170852/plots/fig_3_43/cor_collect/patterncor.png`

Reply:
Thanks @remi-kazeroni for spotting that! I fixed it with adding an optional setting for the diag_script.